### PR TITLE
Balance

### PR DESCRIPTION
**Pull request by godscreator**
Fixes issue #23 - Created a transaction recording page "balance" with profit and loss column

branch name is "balance"

(p.s This is my first time contributing . )

### CODE REVIEW

File: core/templates/core/balance.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    <br>
+    <input type="text" id="market-search" onkeyup="marketSearch()" placeholder="Search for stock names..">
+    <br><br>
+    
+    <table class="table balance-sheet" >
+    	<thead>
+    		<tr>
+    			<th scope="col"> Stock name</th>
+    			<th scope="col"> Status</th>
+    			<th scope="col"> Qty.</th>
```
Comment (ankan17) [inline core/templates/core/balance.html]: Just make it as "Quantity"

File: core/templates/core/includes/navbar.html [hunk 2]
```diff
@@ -23,6 +23,9 @@
                 <li class="nav-item {% if current == 'Market' %}active{% endif %}">
                     <a class="nav-link" href="/market">Market</a>
                 </li>
+                <li class="nav-item {% if current == 'Balance' %}active{% endif %}">
+                    <a class="nav-link" href="/balance">Balance</a>
```
Comment (ankan17) [inline core/templates/core/includes/navbar.html]: I think it's better to name this feature (and hence, the tab) as "Transactions". Please change balance to transactions at all places.

**Review by ankan17 — CHANGES_REQUESTED** [review]
Thanks a lot for the PR. Looks just about right. However, there are a few minor modifications required.

**Comment by godscreator** [comment]
Ok

**Comment by godscreator** [comment]
made changes as requested

**Comment by godscreator** [comment]
Made changes as requested

**Review by ankan17 — APPROVED** [review]

**Comment by godscreator** [comment]
Thanks for merging 😁

**Pull request merged**